core/board/board.go

```diff
@@ -31,18 +31,33 @@ func NewBoard(size int) *Board {
 }
 
 // AddStone adds a stone to the board.
+// Does not remove captured stones.
```

Comment:
I don't think we should have AddStone and PlaceStone. Just PlaceStone is sufficient -- by default captures should always be removed.